custom/abdm/milestone_one/utils/decorators.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from functools import wraps
+from typing import List
+
+from custom.abdm.milestone_one.utils.response_util import (
+    generate_invalid_req_response,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def required_request_params(params):
+    """
+    Checks if the parameters provided in the decorator(a list of strings) are present in the DRF request.
+    If not, raises 400 Bad Request error.
+    """
+
+    def decorate(fn):
+        if not (params and isinstance(params, List)):
+            error_msg = "Request could not be validated as a valid input not provided. \
+                Required: List of parameters."
+            logger.warning(error_msg)
```

Comment:
Do you still need this? I remember we offlined about retaining logging for debugging purposes.

Reply:
https://github.com/dimagi/commcare-hq/pull/32332/commits/4a9020c2c21ca0d02451817e02798e183fa09ee0